std/file.d

```diff
@@ -4444,6 +4445,7 @@ void rmdirRecurse(scope const(char)[] pathname)
 }
```

Comment:
`rmdirRecurse(DirEntry(() @trusted {return cast(string) pathname}()));`

Reply:
Please address this and mark `void rmdirRecurse(scope const(char)[] pathname)` `@safe`

Reply:
Done